### PR TITLE
fix #1675: Improve check for block(s) removal

### LINKED ISSUE #1675: Improve check for block(s) removal in modtool_rm.py
[here](https://github.com/gnuradio/gnuradio/blob/master/gr-utils/python/modtool/modtool_rm.py#L51) the code is accepting empty spaces as argument but this shouldn't be the case as such block cannot exist.

### PR DESCRIPTION
**Pull request by swap-nil7**
Proposed fix for improving blocks(s) removal in modtool_rm.py

### CODE REVIEW

**Comment by swap-nil7** [comment]
@noc0lour @marcusmueller can you review please?

File: gr-utils/python/modtool/modtool_rm.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def setup(self, options, args):
         else:
             with SequenceCompleter():
                 self._info['pattern'] = raw_input('Which blocks do you want to delete? (Regex): ')
-        if len(self._info['pattern']) == 0:
+        if(not self._info['pattern'] or self._info['pattern'].isspace()):
```
Comment (marcusmueller) [inline gr-utils/python/modtool/modtool_rm.py]: please, no `()` around `if` conditions – this isn't C.

**Comment by marcusmueller** [comment]
@noc0lour BB hiccup^^

**Comment by swap-nil7** [comment]
@marcusmueller can you please review now? :)

**Comment by mbr0wn** [comment]
Git commit comment applies to other PRs too.

Thanks for submitting! This is a nice cleanup.

**Comment by swap-nil7** [comment]
@mbr0wn: changed the commit message. :)

**Comment by marcusmueller** [comment]
Woohoo, @swap-nil7, we're on a merge run!

**Pull request merged**